addons/alerting.libsonnet

```diff
@@ -70,13 +70,18 @@ function(config) {
   local infoRoute = trimRoutesTmpl(std.manifestYamlDoc(infoRouteTmpl, quote_keys=false)),
   local allRoutes = std.stripChars(criticalRoute + '\n' + teamRoutes + '\n' + warningRoute + '\n' + infoRoute, '[]'),
 
-  local slackReceiver(name, channel, webhook) = {
+  local slackReceiver(name, channel) = {
     name: name,
     slack_configs: [
       {
         send_resolved: true,
-        api_url: webhook,
+        api_url: 'https://slack.com/api/chat.postMessage',
         channel: channel,
+        http_config: {
+          authorization: {
+            credentials: config.alerting.clusterMonitoringOAuthToken,
```

Comment:
Could we change the field to something that resembles slack somehow?

maybe `slackOAuthToken`? Or something similar to that? 🙂